projects/packages/videopress/src/client/admin/components/video-row/index.tsx

```diff
@@ -0,0 +1,263 @@
+
+const millisecondsToMinutesAndSeconds = ( milliseconds: number ) => {
+	const minutes = Math.floor( milliseconds / 60000 );
+	const seconds = Math.floor( ( milliseconds % 60000 ) / 1000 );
+	return `${ minutes }:${ seconds < 10 ? '0' : '' }${ seconds }`;
+};
+
+const PopoverWithAnchor = ( {
+	anchorRef,
+	children = null,
+}: {
+	anchorRef: HTMLElement | null;
+	children: React.ReactNode;
+} ) => {
+	if ( ! anchorRef ) {
+		return null;
+	}
+
+	return (
+		<Popover placement="top" offset={ 15 } noArrow={ false } anchorRef={ anchorRef }>
+			<Text variant="body-small" className={ styles.popover }>
+				{ children }
+			</Text>
+		</Popover>
+	);
+};
+
+const ActionItem = ( {
+	icon,
+	children,
+	className = '',
+}: {
+	icon: JSX.Element;
+	children: React.ReactNode;
+	className?: string;
+} ) => {
+	const [ anchorRef, setAnchorRef ] = useState( null );
+	const [ showPopover, setShowPopover ] = useState( false );
+
+	return (
+		<div
+			ref={ setAnchorRef }
+			onMouseOver={ () => setShowPopover( true ) }
+			onMouseLeave={ () => setShowPopover( false ) }
+			onFocus={ () => setShowPopover( true ) }
+			onBlur={ () => setShowPopover( false ) }
+			className={ className }
+		>
+			<Icon icon={ icon } />
+			{ showPopover && <PopoverWithAnchor anchorRef={ anchorRef }>{ children }</PopoverWithAnchor> }
+		</div>
+	);
+};
+
+const QuickActions = ( { button }: { button: React.ReactNode } ) => {
+	// Hiding it based on Design request:
+	// https://github.com/Automattic/jetpack/issues/25742#issuecomment-1223123815
+	const HIDE_QUICK_ACTIONS = false;
+
+	if ( HIDE_QUICK_ACTIONS ) {
+		return null;
+	}
+
+	return (
+		<div className={ styles.actions }>
+			{ button }
+			<ActionItem icon={ image }>{ __( 'Update thumbnail', 'jetpack-videopress-pkg' ) }</ActionItem>
+			<ActionItem icon={ privacy }>{ __( 'Update privacy', 'jetpack-videopress-pkg' ) }</ActionItem>
+			<ActionItem icon={ trash } className={ styles.trash }>
+				{ __( 'Delete video', 'jetpack-videopress-pkg' ) }
+			</ActionItem>
+		</div>
+	);
+};
+
+const Stats = ( {
+	duration,
+	uploadDate,
+	plays,
+	isPrivate,
+}: {
+	duration: string;
+	uploadDate: string;
+	plays: number;
+	isPrivate: boolean;
+} ) => {
+	const [ isSmall ] = useBreakpointMatch( 'sm' );
+	const durationLabel = __( 'Duration', 'jetpack-videopress-pkg' );
+	const playsLabel = __( 'Plays', 'jetpack-videopress-pkg' );
+	const privacyLabel = __( 'Privacy', 'jetpack-videopress-pkg' );
+
+	return (
+		<div className={ classNames( styles.stats, { [ styles.small ]: isSmall } ) }>
+			<div className={ styles.privacy } aria-disabled={ isSmall ? 'false' : 'true' }>
+				{ isPrivate && ! isSmall && <Icon icon={ privacy } /> }
+				{ isSmall && (
+					<>
+						<span>{ privacyLabel }</span>
+						<span>
+							{ isPrivate
+								? __( 'Private', 'jetpack-videopress-pkg' )
+								: __( 'Public', 'jetpack-videopress-pkg' ) }
+						</span>
+					</>
+				) }
+			</div>
+			<div className={ styles.duration }>
```

Comment:
`duration` CSS class is not defined.

Reply:
We should replace this div by a Text:

```suggestion
			<Text variant="body" className={ styles.duration }>
```

Reply:
Forgot to remove them, working on it

Reply:
Done https://github.com/Automattic/jetpack/pull/25798/commits/5a88689a0ba77c6cf2f833b44f43f0515dca5a99